packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/programmableKeys/telecom-telephony-line-phone-programmableKeys.controller.js

```diff
@@ -97,7 +97,14 @@ export default class TelecomTelephonyLinePhoneProgammableKeysCtrl {
     return this.line.getPhone().then(() => {
       if (this.line.hasPhone) {
         return this.line.phone.initDeffered().then(() => {
-          this.functionKeys.raw = cloneDeep(this.line.phone.functionKeys);
+          this.functionKeys.raw = this.line.phone.functionKeys.map((functionKey) => {
```

Comment:
It will be in conflict with: https://github.com/ovh/manager/pull/1685/files#diff-6a5954edbbe736a87468cb0d7e25787dR101-R111.

Original Pull Request: #1685

Is it intentional?

Reply:
> Regarding translations update you can refer to the [CONTRIBUTING.md](https://github.com/ovh/manager/blob/master/CONTRIBUTING.md#translations) file. Updating only `Messages_fr_FR.json` files is totally fine.

Ok, understood

Reply:
> It will be in conflict with: https://github.com/ovh/manager/pull/1685/files#diff-6a5954edbbe736a87468cb0d7e25787dR101-R111.
> 
> Original Pull Request: #1685
> 
> Is it intentional?

I wanted to code both features, so I will update the second pull request once the first is merged in develop branch.
You just have to decide which PR you will accept first ;)

Reply:
Updated PR against head of develop